modules/apps/map/map-openstreetmap/src/main/resources/META-INF/resources/js/MapOpenStreetMap.es.js

```diff
@@ -48,13 +48,10 @@ class MapOpenStreetMap extends MapBase {
 
 		super(args);
 
-		const {boundingBox, tileURI = defaultTileURI} = args;
+		const {tileURI = defaultTileURI} = args;
 
-		this._STATE_ = {
-			tileURI,
-		};
+		this._STATE_.tileURI = tileURI;
```

Comment:
Oh since `this._STATE_` is already defined in the parent class, you'll probably want to do something like

```js
this._STATE_ = {
	...this._STATE_,
	tileURI: tileURI
}
```

Reply:
By doing this you are essentially saying "_STATE_ has another item called tileURI", whereas if you just have `this._STATE_.tileURI` it is hard to know if that initial value comes from the parent class or if it is a new item

Reply:
Sure, but what's the difference between my solution and your proposal? Is there a quantifiable difference or just stylistic preference?